CheckYourEligibility-Parent/Controllers/HomeController.cs

```diff
@@ -37,5 +56,48 @@ public IActionResult Parental_Guidance()
         {
             return View();
         }
+
+        [HttpGet]
+        public async Task<IActionResult> SchoolList()
+        {
+            var schoolList = await _parentService.GetSchool("school");
+
+            // Check if schoolList or schoolList.Data is null
+            var schools = schoolList?.Data?.ToList() ?? new List<CheckYourEligibility.Domain.Responses.Establishment>();
+
+            var viewModel = new SchoolListViewModel
+            {
+                Schools = schools,
+                IsRadioSelected = null // Default value to ensure radio buttons are blank
+            };
+
+            return View(viewModel);
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> SchoolList(SchoolListViewModel viewModel)
+        {
+            if (viewModel.IsRadioSelected.HasValue)
+            {
+                if (viewModel.IsRadioSelected == true)
+                {
+                    return RedirectToAction("Enter_Details", "Check");
+                }
+                else
+                {
+                    return Redirect("https://www.gov.uk/apply-free-school-meals"); // this could be in appsettings instead
```

Comment:
Is this part of the ticket? Seems strange to redirect them off service for all environments.

Reply:
There was a bit of confusion in the ticket as the tickets AC use this url but there did exist a page saying "you are not able to use the service" within prototype, had a call with Adam about it and he said to used this link